sys-cluster/keepalived/keepalived-2.3.1.ebuild

```diff
@@ -36,10 +36,6 @@ DOCS=(
 	doc/keepalived.conf.SYNOPSIS doc/NOTE_vrrp_vmac.txt
 )
 
-PATCHES=(
-	"${FILESDIR}"/${P}-libressl.patch #903001
-)
-
```

Comment:
Since you are dropping the last version which needed this patch, you should delete the patch too. :)

Reply:
Good point, patch dropped, thanks!